FileWatcherImplementation does not contain a definition for DirectWindowsApi

Hey, on Ubuntu 22.04, Unity 2023.2.5f1.

When importing through .git url i get this error.
Found a previous closed issue regarding the FileWatcher but couldn't quite understand
if this is related to that or not..

Library/PackageCache/com.handzlikchris.fastscriptreload@6412e8454a/Scripts/Editor/FastScriptReloadManager.cs(160,48): error CS0117: 'FileWatcherImplementation' does not contain a definition for 'DirectWindowsApi'

--- a/Assets/Scripts/Editor/FastScriptReloadManager.cs
+++ b/Assets/Scripts/Editor/FastScriptReloadManager.cs
@@ -157,7 +157,7 @@ namespace FastScriptReload.Editor
                     _fileWatchers.Add(fileWatcher);
                     
                     break;
-#if UNITY_2021_1_OR_NEWER
+#if UNITY_2021_1_OR_NEWER && UNITY_EDITOR_WIN
                 case FileWatcherImplementation.DirectWindowsApi: 
                 // On Windows, this is a WindowsFileSystemWatcher.
                 // On other platforms, it's the default Mono implementation.

